Document/0x07a-Testing-Authentication-and-Session-Management.md

```diff
@@ -462,100 +462,71 @@ Most of the frameworks have a parameter to configure the session timeout. This p
 * [2] OWASP Session management cheatsheet https://www.owasp.org/index.php/Session_Management_Cheat_Sheet
 
 
-### Testing 2-Factor Authentication
+### Testing 2-Factor Authentication and Step-up Authentication
 
 #### Overview
 
-https://authy.com/blog/security-of-sms-for-2fa-what-are-your-options/
--- TODO [Provide a general description of the issue "Testing 2-Factor Authentication".] --
+Two-factor (2FA) is becoming a standard when logging into mobile apps. Typically the first factor might be credentials (username/password), followed by a second factor which could be an One Time Password (OTP) sent via SMS. It could be implemented also through different mechanisms, that use factors like:
+* Passwords
```

Comment:
It would be nice to take another approach here: what is the second factor in terms of its nature? (E.g. something you have/know/are/where you are and then relate to the possible items here)